app/api/events.py

```diff
@@ -128,19 +183,29 @@ def after_create_object(self, event, data, view_kwargs):
         uer = UsersEventsRoles(user, event, role)
         save_to_db(uer, 'Event Saved')
 
-    decorators = (accessible_events, )
+    def before_create_object(self, data, view_kwargs):
+        data['starts-at'] = data['starts-at'].replace(tzinfo=None)
+        data['ends-at'] = data['ends-at'].replace(tzinfo=None)
+
+    def after_get(self, result):
+        result = changeToTimezone(result)
+
+    def after_post(self, result):
+        result = changeToTimezone(result)
+
+    decorators = (jwt_required,)
```

Comment:
@poush 
Are you sure ?
cause now i am getting some other errors and am unable to even fetch events.